src/codeflare_sdk/ray/cluster/config.py

```diff
@@ -115,6 +121,7 @@ class ClusterConfiguration:
     min_memory: Optional[Union[int, str]] = None  # Deprecating
     max_memory: Optional[Union[int, str]] = None  # Deprecating
     num_gpus: Optional[int] = None  # Deprecating
+    tolerations: Optional[List[V1Toleration]] = None
```

Comment:
```suggestion
    worker_tolerations: Optional[List[V1Toleration]] = None
```
For the sake of consistency and minimizing confusion we should rename to `worker_tolerations`